IPIP/0001-lightweight-improvement-proposal-process.md

```diff
@@ -54,29 +59,33 @@ document** based on the template in [`ipfs/specs/IPIP/0000-template.md`](./0000-
 When a new specification file is added to the repo, it should be based on
 the template at [`ipfs/specs/template.md`](../template.md).
 
-### Reviewing IPIPs
-
-[Specs Stewards] will review new IPIP PRs during periodical (best-effort) triage.
+When naming a new proposal, don't try to introduce an IPIP number; we will do that only for
+IPIPs that are approved before accepting into master.
 
-IPFS Community is encouraged to participate in the review process.
+Proposals are officially submitted when a pull request into master is opened
 
-IPIP can be either:
-- merged,
-- rejected (PR close without merging),
-- deferred (converting PR back to a draft).
+Proposals that were reviewed and rejected will be moved into /archive folder and then merged into master
 
-The final decision belongs to [Specs Stewards].
-
-### Merging IPIPs
-
-PR with a IPIP can be merged only after two [Specs Stewards] approve it and
-there are no objections from other Stewards.
+### Reviewing IPIPs
 
-IPIP number is assigned before the PR merge.
+1. [Specs Stewards] will do an initial triage of newly opened PRs roughly monthly. They'll try to filter out
+noise, so that community consideration is given only to reasonable proposals; others they'll reject.
+2. Specs Stewards will post to the forums linking to the proposal; directing feedback/discussion to
```

Comment:
What do we mean by "the forums"?
Should I create "IPIP" category under https://discuss.ipfs.tech/c/protocol/30 ?